test/shed_functional/functional/test_1000_install_basic_repository.py

```diff
@@ -21,8 +21,9 @@ def test_0000_initiate_users( self ):
 
     def test_0005_ensure_repositories_and_categories_exist( self ):
         '''Create the 0000 category and upload the filtering repository to it, if necessary.'''
-        category = self.create_category( name='Test 0000 Basic Repository Features 1', description='Test 0000 Basic Repository Features 1' )
-        self.create_category( name='Test 0000 Basic Repository Features 2', description='Test 0000 Basic Repository Features 2' )
+        self.login( email=common.admin_email, username=common.admin_username )
+        category = self.create_category( name='Test 0000 Basic Repository Features 1', description='Test Description 0000 Basic Repository Features 1' )
+        category = self.create_category( name='Test 0000 Basic Repository Features 2', description='Test Description 0000 Basic Repository Features 2' )
```

Comment:
The added `category = ` at the start of this line changes the category in which the repository 'filtering_0000' is created. I guess that the test `test_0015_browse_test_0000_category()` below does not fail only because a repository of the same name is created in category 'Test 0000 Basic Repository Features 1' during a previous test.